deploy/template.env

```diff
@@ -0,0 +1,37 @@
+# ##############################################################################
+# Wikibase Suite
+#
+# This file is a template for the .env file used by the Wikibase Suite Docker
+# Compose setup.
+#
+# WARNING: Do not add comments on the same line as env vars, as in some
+# environments they will be included in the var!
+#
+# shellcheck disable=2034 # do not warn about unused variables
```

Comment:
I know you had commented on this before, but just checking: is there no place we can configure this exception exception inline?

Reply:
Yeah, that probably really a config that belongs on my machine.

I should remove it from the file, right?

Reply:
I think so, yes, but Spellcheck is part of our linting chain so think it will complain without it here or us disabling the rule elsewhere?

Reply:
done